docs/docs/display_conditions.md

```diff
@@ -0,0 +1,499 @@
+
+displayConditions: {
+  date: {
+    start_at: "2017-02-12T11:00:00.000Z",
+    end_at: "2017-03-12T11:00:00.000Z"
+  }
+}
+```
+  
+## impressions
+Hide the module after a certain number of impressions
+
+<table>
+  <tr>
+    <th>VALUE</th>
+    <th>TYPE</th>
+    <th>BEHAVIOR</th>
+  </tr>
+  <tr>
+    <td>impressions</td>
+    <td>obj</td>
+    <td>must be object formatted using following options / values</td>
+  </tr>
+</table>
+
+<table>
+  <tr>
+    <td colspan="3" align="center"><code>date</code> object</td>
+  </tr>
+  <tr>
+    <th>VALUE</th>
+    <th>TYPE</th>
+    <th>BEHAVIOR</th>
+  </tr>
+  <tr>
+    <td> session </td>
+    <td>int</td>
+    <td><code>optional</code> integer count of how many session-based impressions before showing the module</td>
+  </tr>
+  <tr>
+    <td> total </td>
+    <td>int</td>
+    <td><code>optional</code> integer count of how many total (multisession) impressions before showing the module</td>
+  </tr>
+</table>
+
+``` javascript
+// example: hide module after the second impressions in the same session
+
+displayConditions: {
+  impressions: {
+    session: 2
+  }
+}
+```
+
+``` javascript
+// example: hide module after five total impressions
+
+displayConditions: {
+  impressions: {
+    total: 5
+  }
+}
+```   
+
+``` javascript
+// example: hide the module after the second impression in the same session
+// or if it has been seen five times ever
+
+displayConditions: {
+  impressions: {
+    session: 2,
+    total: 5
+  }
+}
+```
+  
+  
+## hideAfterAction
+Hide the module after a particular action has been taken (`closed, confirm, cancel`)
+
+<table>
+  <tr>
+    <th>VALUE</th>
+    <th>TYPE</th>
+    <th>BEHAVIOR</th>
+  </tr>
+  <tr>
+    <td>hideAfterAction</td>
+    <td>obj</td>
+    <td>must be object formatted using following options / values</td>
+  </tr>
+</table>
+
+<table>
+  <tr>
+    <td colspan="3" align="center"><code>closed</code> / <code>confirm</code> / <code>cancel</code> object</td>
+  </tr>
+  <tr>
+    <th>VALUE</th>
+    <th>TYPE</th>
+    <th>BEHAVIOR</th>
+  </tr>
+  <tr>
+    <td>hideCount</td>
+    <td>int</td>
+    <td><code>optional</code> integer count of times module has been closed manually by user</td>
+  </tr>
+  <tr>
+    <td>duration</td>
+    <td>int</td>
+    <td><code>optional</code> integer representing how long the module should be hidden in seconds</td>
+  </tr>
+</table>
+
+
+``` javascript
+// example: hide module for 6 minutes after 5th close
+
+displayConditions: {
+  hideAfterAction: {
+    closed: {
+      hideCount: 5,
+      duration: 360
+    }
+  }
+}
+```
+
+``` javascript
+// example: hide module permanently after confirmation has been clicked
+
+displayConditions: {
+  hideAfterAction: {
+    confirm: {
+      hideCount: 1
+    }
+  }
+}
+```
+
+``` javascript
+// example: hide module for 1 week after cancel has been clicked
+
+displayConditions: {
+  hideAfterAction: {
+    cancel: {
+      hideCount: 1,
+      duration: 604800
```

Comment:
A common convention is to spell out the multiplication. I think that'll help drive home the fact that these are seconds.

```
duration: 60 * 60 * 24 * 7
```